modules/serverless-gclb/variables.tf

```diff
@@ -34,3 +34,12 @@ variable "notification_channels" {
   type        = list(string)
   default     = []
 }
+
+variable "iap" {
+  description = "IAP configuration for the load balancer."
+  type = object({
+    oauth2_client_id     = string
+    oauth2_client_secret = string
```

Comment:
Is this something we should mark as sensitive?

Reply:
The GCP resource already marks this sensitive in the generated output. I also don't know if we can mark it as sensitive here (https://discuss.hashicorp.com/t/is-it-possible-to-mark-an-attribute-of-an-object-as-sensitive/24649)

```
resource "google_compute_backend_service" "public-services" {
      + connection_draining_timeout_sec = 300
      + creation_timestamp              = (known after apply)
      + fingerprint                     = (known after apply)
      + generated_id                    = (known after apply)
      + id                              = (known after apply)
      + load_balancing_scheme           = "EXTERNAL"
      + name                            = "nami-frontend"
      + port_name                       = (known after apply)
      + project                         = "chainguard-nami"
      + protocol                        = (known after apply)
      + self_link                       = (known after apply)
      + session_affinity                = (known after apply)
      + timeout_sec                     = (known after apply)

      + backend {
          + balancing_mode               = "UTILIZATION"
          + capacity_scaler              = 1
          + group                        = (known after apply)
          + max_connections              = (known after apply)
          + max_connections_per_endpoint = (known after apply)
          + max_connections_per_instance = (known after apply)
          + max_rate                     = (known after apply)
          + max_rate_per_endpoint        = (known after apply)
          + max_rate_per_instance        = (known after apply)
          + max_utilization              = (known after apply)
            # (1 unchanged attribute hidden)
        }

      + iap {
          + oauth2_client_id            = "123526589617-dccnftqc6snrcpa4o4anvm286bsamoe7.apps.googleusercontent.com"
          + oauth2_client_secret        = (sensitive value)
          + oauth2_client_secret_sha256 = (sensitive value)
        }
    }
```